apps/api/src/domain/domainModelValidators/__tests__/domainModelValidators/buildDomainModelValidatorTestCases/bibliographicReferenceInvalidTestCases/bookBibliographicReference.domainModelValidatorTestCases.ts

```diff
@@ -46,8 +59,7 @@ export const buildBookBibliographicReferenceTestCases =
                 data: {
                     ...validDto.data,
                     abstract: '',
-                    // TODO remove cast
-                } as DTO<BookBibliographicReferenceData>,
```

Comment:
Nice. Only use casting as a temporary workaround?

Reply:
Yea. I managed to use a generic and type inference on the helper we used to build the `validDto` to give us type safety here. It cleaned up a lot of casts.